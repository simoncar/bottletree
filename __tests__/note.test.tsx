import React from "react";
import Note from "../app/(app)/note";
import { renderRouter, screen } from "expo-router/testing-library";

it("my-test", async () => {
  const MockComponent = jest.fn(() => <Note />);

  renderRouter(
    {
      index: MockComponent,
      "note?projectId=73JwAXeOEhLXUggpVKK9": MockComponent,
      "(group)/b": MockComponent,
    },
    {
      initialUrl: "/",
    },
  );

  expect(screen).toHavePathname("/");
  expect(screen).toMatchSnapshot();
});
