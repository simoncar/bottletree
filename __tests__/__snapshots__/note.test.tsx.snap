// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my-test 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <RNCSafeAreaView
    edges={
      {
        "bottom": "additive",
        "left": "additive",
        "right": "additive",
        "top": "additive",
      }
    }
    style={
      {
        "flex": 1,
      }
    }
  >
    <RCTSafeAreaView>
      <RCTScrollView>
        <View>
          <View
            style={
              {
                "flex": 1,
                "justifyContent": "center",
                "padding": 20,
              }
            }
          >
            <View
              style={
                {
                  "flex": 1,
                  "justifyContent": "flex-start",
                }
              }
            >
              <TextInput
                autoFocus={true}
                multiline={true}
                numberOfLines={10}
                onChangeText={[Function]}
                placeholder="Add Note"
                style={
                  [
                    {
                      "fontSize": 25,
                    },
                    {
                      "color": "#000",
                    },
                  ]
                }
                value=""
              />
            </View>
          </View>
        </View>
      </RCTScrollView>
    </RCTSafeAreaView>
  </RNCSafeAreaView>
</RNCSafeAreaProvider>
`;
